lib/puppet/parser/functions/convert_to_ensurable.rb

```diff
@@ -0,0 +1,18 @@
+#
+# convert_to_ensurable.rb
+#
+# Simple converter function to map boolean values into their 'ensurable' counterparts, :present & :absent
+#
+module Puppet::Parser::Functions
```

Comment:
So would I and there are many variations of them about.  In fact there is a stdlib function 'bool2str' that would suffice to provide the necessary functionality (though somewhat more hacky)

But doing this would require us to drop support for stdlib < 4.10.0  This might impact end users more than adding the function directly, so I am at a loss for how to proceed.